### PR TITLE
Add 3 transaction related sysvar

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Add 3 transaction related sysvars

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/9897  https://github.com/pingcap/docs/pull/9896 https://github.com/pingcap/docs/pull/9998
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1207,6 +1207,16 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
     - `sql`：当前查询对应的 SQL 语句
 
+### `tidb_guarantee_linearizability` <span class="version-mark">从 v5.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔值
+- 默认值：`OFF`
+- 此变量控制异步提交 (Async Commit) 中提交时间戳的计算方式。默认情况下（使用 `OFF` 值），两阶段提交从 PD 服务器请求一个新的时间戳，并使用该时间戳计算最终提交的时间戳，这样可保证所有并发事务的线性化。
+- 如果将该变量值设为 `ON`，从 PD 获取的时间戳会被保存下来，这种情况下只保证因果一致性但不保证线性一致性。详情请参考博文 [Async Commit 原理介绍](https://pingcap.com/zh/blog/async-commit-principle)。
```
Comment (Oreoxmt) [inline system-variables.md:1217]:
```suggestion
- 如果将该变量值设为 `ON`，从 PD 获取的时间戳会被保存下来，这种情况下只保证因果一致性但不保证线性一致性。详情请参考 PingCAP 博文 [Async Commit 原理介绍](https://pingcap.com/zh/blog/async-commit-principle)。
```

File: system-variables.md [hunk 2]
```diff
@@ -1318,6 +1328,18 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+
+- 作用域：SESSION
+- 是否持久化到集群：否
+- 类型：字符串
+- 此变量用于获取当前会话中最后一个事务的信息。这是一个只读变量。事务信息包括：
+    - 事务的范围
+    - 开始时间戳和提交时间戳
+    - 事务的提交模式，可能是两阶段提交，一阶段提交，或者异步提交
+    - 事务从异步提交或一阶段提交到两阶段提交的回退信息。
+    - 遇到的错误。
```
Comment (Oreoxmt) [inline system-variables.md:1341]:
```suggestion
    - 事务从异步提交或一阶段提交到两阶段提交的回退信息
    - 遇到的错误
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: system-variables.md [hunk 3]
```diff
@@ -1207,6 +1207,16 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
     - `txn_mode`：事务模式。可选值：`OPTIMISTIC`（乐观事务模式），或 `PESSIMISTIC`（悲观事务模式）
     - `sql`：当前查询对应的 SQL 语句
 
+### `tidb_guarantee_linearizability` <span class="version-mark">从 v5.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔值
+- 默认值：`OFF`
+- 此变量控制异步提交 (Async Commit) 中提交时间戳的计算方式。默认情况下（使用 `OFF` 值），两阶段提交从 PD 服务器请求一个新的时间戳，并使用该时间戳计算最终提交的时间戳，这样可保证所有并发事务的线性化。
```
Comment (cfzjywxk) [inline system-variables.md:1216]:
```suggestion
- 此变量控制异步提交 (Async Commit) 中提交时间戳的计算方式。默认情况下（使用 `OFF` 值），两阶段提交从 PD 服务器请求一个新的时间戳，并使用该时间戳计算最终提交的时间戳，这样可保证所有并发事务可线性化。
```

File: system-variables.md [hunk 4]
```diff
@@ -1737,6 +1759,16 @@ explain select * from t where age=5;
...
 - 如果读写冲突较为严重，开启此功能会增加额外开销和延迟，造成性能回退。更详细的说明，请参考[读已提交隔离级别 (Read Committed) 文档](/transaction-isolation-levels.md#读已提交隔离级别-read-committed)。
 
+### `tidb_read_consistency` <span class="version-mark">New in v5.4.0</span>
+
+- 作用域：SESSION
+- 是否持久化到集群：否
+- 类型：字符串
+- 默认值：`strict`
+- 此变量用于控制自动提交的读语句的读一致性。
+- 如果将变量值设置为 `weak`，则直接跳过读语句遇到的锁，读的执行可能会更快，这就是弱一致性读模式。但在该模式下，事务语义（例如原子性）和分布式一致性（例如线性化）并不能得到保证。
```
Comment (cfzjywxk) [inline system-variables.md:1769]:
```suggestion
- 如果将变量值设置为 `weak`，则直接跳过读语句遇到的锁，读的执行可能会更快，这就是弱一致性读模式。但在该模式下，事务语义（例如原子性）和分布式一致性（线性一致性）并不能得到保证。
```

File: system-variables.md [hunk 5]
```diff
@@ -1207,6 +1207,16 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
     - `sql`：当前查询对应的 SQL 语句
 
+### `tidb_guarantee_linearizability` <span class="version-mark">从 v5.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔值
+- 默认值：`OFF`
+- 此变量控制异步提交 (Async Commit) 中提交时间戳的计算方式。默认情况下（使用 `OFF` 值），两阶段提交从 PD 服务器请求一个新的时间戳，并使用该时间戳计算最终提交的时间戳，这样可保证所有并发事务的线性化。
+- 如果将该变量值设为 `ON`，从 PD 获取的时间戳会被保存下来，这种情况下只保证因果一致性但不保证线性一致性。详情请参考 PingCAP 博文 [Async Commit 原理介绍](https://pingcap.com/zh/blog/async-commit-principle)。
```
Comment (cfzjywxk) [inline system-variables.md:1217]:
```suggestion
- 如果将该变量值设为 `ON`，从 PD 获取的时间戳的操作会被省掉，这种情况下只保证因果一致性但不保证线性一致性。详情请参考 PingCAP 博文 [Async Commit 原理介绍](https://pingcap.com/zh/blog/async-commit-principle)。
```

Comment (cfzjywxk) [reply system-variables.md:1217]: 这里英文的地方可能表述也不是特别准确，是这个操作会被节省掉或者跳过。

Comment (TomShawn) [reply system-variables.md]: @cfzjywxk 英文改过来了：https://github.com/pingcap/docs/pull/9998

**Review by cfzjywxk — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**